scenario.py

```diff
@@ -103,6 +104,12 @@ def __init__(self, params, experiment_path):
         else:
             self.minibatch_count = 20
 
+        if "fit_batches_count" in params:
+            self.fit_batches_count = params["fit_batches_count"]
+            assert self.fit_batches_count > 0
```

Comment:
so this the number of gradient update per mini batch ?
Could we call it `gradient_updates_per_minbach_count `? It's a bit long, but it's hard to guess what is `fit_batches_count`

Reply:
I propose to keep it that way, with its definition + explicative schema in the `README.md`, since it resonates with `batch_size` and is specific to usage of `fit()`. If in a few weeks it still looks odd we'll change it.